test/lib/ui5framework/Sapui5Resolver.js

```diff
@@ -0,0 +1,103 @@
+const test = require("ava");
+const sinon = require("sinon");
+const path = require("path");
+
+const Sapui5Resolver = require("../../../lib/ui5Framework/Sapui5Resolver");
+
+test.serial("Sapui5Resolver: loadDistMetadata loads metadata once from @sapui5/distribution-metadata package", async (t) => {
+	const resolver = new Sapui5Resolver({
+		cwd: "/test-project/",
+		version: "1.75.0"
+	});
+
+	const getTargetDirForPackage = sinon.stub(resolver._installer, "_getTargetDirForPackage");
+	getTargetDirForPackage.withArgs({
```

Comment:
What happens if the stub is called with the wrong arguments? Without `callsFake` and an exception like some other tests, [my understanding](https://sinonjs.org/releases/v7.5.0/stubs/) is that it returns `undefined`. I fear that this might make it pretty hard to understand what makes this test fail in such a case.

Maybe add a `callsFake` with an exception or assert what the stub is being called with?